pkgs/applications/editors/vscode/with-extensions.nix

```diff
@@ -51,12 +51,10 @@ let
     paths = vscodeExtensions;
   };
 
-  wrappedExeName = "code";
-  exeName = wrappedExeName;
-
-  wrapperExeFile = writeScript "${exeName}" ''
+  inherit (vscode) executableName;
+  wrapperExeFile = writeScript "${executableName}" ''
```

Comment:
I think this can be refactored to use `makeWrapper`. This is just an idea in case somebody cares to implement.

Reply:
Done, I think I did it correctly.